models/balancer/ethereum/balancer_v1_ethereum_trades.sql

```diff
@@ -0,0 +1,99 @@
+{{ config(
+    schema='balancer_v1_ethereum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
we should likely remove the post-hook on version level spells, as there is a minor bug in the UI data explorer at this time. if you look closely, you'll notice both version spells and the rollup spell all have same properties passed in this macro, leading to confusion on the explorer of which table to display. in the future, we will look to supply additional parameters to separate the spells

Reply:
I see. So we only leave the post-hook in the roll-up spell for now.